modules/@apostrophecms/ui/ui/apos/components/AposFilterMenu.vue

```diff
@@ -14,6 +14,7 @@
           :icon="
             set.field.type === 'select' ? 'unfold-more-horizontal-icon' : ''
           "
+          :no-blur-emit="true"
```

Comment:
a prop I added in the past to avoid emitting `validateAndEmit` on blur (avoid refetching all images when clicking outside of filters our search for example..)